ckanext/dia_theme/templates/footer.html

```diff
@@ -2,36 +2,32 @@
     <div class="container">
         <div class="row">
 	    <div class="span3">
-		<h2 class="footcolheader">About us</h2>
+		<h2 class="footcolheader">Get in touch</h2>
```

Comment:
I think we'll add a link to the "Request dataset" here so that there is still a global place to get to there. @matt-in-a-hat if you can add this back in.

I wont be merging this until after the docker production deploy though so I'll approve for merge but we'll wait till after Tues next week to actually add to the lice site.